nemo/collections/nlp/models/language_modeling/transformer_lm_model.py

```diff
@@ -74,7 +74,12 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         model_name = encoder_cfg_dict.pop('model_name', None)
         pretrained = encoder_cfg_dict.pop('pretrained', False)
         self.encoder = get_transformer(
-            library=library, model_name=model_name, pretrained=pretrained, config_dict=encoder_cfg_dict, encoder=True,
+            library=library,
+            model_name=model_name,
+            pretrained=pretrained,
+            config_dict=encoder_cfg_dict,
+            encoder=True,
+            pre_ln_final_layer_norm=encoder_cfg_dict.get('pre_ln_final_layer_norm', False),
```

Comment:
This looks like a potential source of bugs, shouldn't it be: 
```
pre_ln_final_layer_norm=encoder_cfg_dict.get('pre_ln_final_layer_norm', encoder_cfg_dict.get('pre_ln', True))
```

Reply:
Yes, this looks more correct than what I have right now. It is not a problem right now because even if `pre_ln_final_layer_norm` is True and `pre_ln` is False, the extra layer norm won't be created. I'll make this change.

Reply:
This should now be fixed.